06.wc/mywc.rb

```diff
@@ -61,11 +61,16 @@ def build_total_numbers(count_data_list)
 
 def display_results(count_data_list, option)
   count_data_list.each do |result|
-    print result[:line].to_s.rjust(8) if option['l']
-    print result[:word].to_s.rjust(8) if option['w']
-    print result[:byte].to_s.rjust(8) if option['c']
-    puts " #{result[:file]}"
+    result_to_display = result.map { |key, val| [key, judge_value(key, val)] }.to_h
+    print result_to_display[:line] if option['l']
+    print result_to_display[:word] if option['w']
+    print result_to_display[:byte] if option['c']
+    puts " #{result_to_display[:file]}"
```

Comment:
僕の実装イメージはこんな感じでした。

```suggestion
    print format_value(result[:line]) if option['l']
    print format_value(result[:word]) if option['w']
    print format_value(result[:byte]) if option['c']
    puts " #{result[:file]}"
```

judge_valueの中でfileかどうかを判定するというのはちょっと再利用性が低い（この処理に特化されすぎている）かな〜と思います。
なので「シンプルに与えられた値を8桁で右寄せする」というメソッドにしておく方がベターです。

Reply:
詳細にご教示いただきありがとうございます！！

>再利用性が低い（この処理に特化されすぎている）
間違いないです＞＜この視点が欠けていました！

頂戴したイメージを元に修正いたしました🙇‍♀️🙇‍♀️